blockchain/chainstore.go

```diff
@@ -408,19 +408,27 @@ func (c *ChainStore) GetTxReference(tx *Transaction) (map[*Input]*Output, error)
 	if tx.TxType == RegisterAsset {
 		return nil, nil
 	}
+	transactionCache := make(map[Uint256][]*Output)
```

Comment:
txOutputs is better, transactionCache is miss leading.

Reply:
fixed